src/community/__stories__/data-card-advanced-story.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
+import ResponsiveLayout from '../../responsive-layout';
+import Box from '../../box';
+import {ButtonLink, ButtonPrimary} from '../../button';
+import Tag from '../../tag';
+import DataCardAdvanced from '../../data-card-advanced';
+import imgExample from '../../__stories__/images/avatar.jpg';
+import Image from '../../image';
+import StackingGroup from '../../stacking-group';
+
+import type {TagType} from '../../tag';
+
+export default {
+    title: 'Community/DataCardAdvanced',
+};
+
+type Args = {
```

Comment:
I think this is my last comment in this review, I promise. 

I think would be nice to add a boolean control to show "extra" like in data card story.

You can include inside extra a <Placeholder /> component.
https://mistica-web.vercel.app/?path=/story/components-cards-data-card--default
![image](https://github.com/Telefonica/mistica-web/assets/6722153/55a34546-f320-4198-8274-35adcfcc6699)

Reply:
kk, understood and well Unfortunately I think this may take a while, it turns out that the data-card-advanced will only be able to receive the PR components of the slots/blocks, until it has the merge of them in the master it will not be able to make the change, but as soon as they go up I will make this change :)

Reply:
I don't understand the relationship with that PR, just include a control to show or hide "extra" prop. You can include inside `extra` something like this https://tinyurl.com/2o2ewpkl, (like the rest of story component with extra prop)

Extra prop is just a area to include any thing, you can include Blocks component inside or just any atomic composition

Reply:
It's just that the data-card-advanced was thought to only allow specific slots that are the PR of the advanced-slot, because of that, however much it is accepted in the playroom, for the storybook it is a different case, when trying this way that it passed in the storybook the page hangs and gives no response:

<img width="960" alt="Captura de tela 2023-06-23 170427" src="https://github.com/Telefonica/mistica-web/assets/123959276/89da9b25-0606-4b33-8040-c01465b20934">

Reply:
oh! I didn't know that! Is there a rationale for this? I don't think we should have that limitation on the DataCardAdvanced extra, the extra should be able to support any component.

@atabel are you agree?

Reply:
Well the data-card-advanced was thought to use only the slots/blocks from the PR of advacend-slots, unfortunately I don't remember exactly why it was thought to be like that, so these two PR's are a set so to speak